folium/folium.py

```diff
@@ -8,6 +8,7 @@
 from __future__ import (absolute_import, division, print_function)
 
 import os
+import tempfile
```

Comment:
I merged in your changes but I'm not sure where this stickler issue came from.

Reply:
It seems git got a bit screwed up. I'm no git wizard so I can't tell exactly what went wrong. I did a rebase, so it seems my branch is now okay. Can you try to merge again, or otherwise rebase as well?